core/field_dropdown.ts

```diff
@@ -644,10 +650,21 @@ export class FieldDropdown extends Field<string> {
     const option = this.selectedOption[0];
     if (isImageProperties(option)) {
       return option.alt;
-    } else if (option instanceof HTMLElement) {
+    } else if (
+      typeof HTMLElement !== 'undefined' &&
+      option instanceof HTMLElement
+    ) {
       return option.title ?? option.ariaLabel ?? option.innerText;
+    } else if (typeof option === 'string') {
+      return option;
     }
-    return option;
+
+    console.warn(
+      "Can't get text for existing dropdown option. If " +
+        "you're using HTMLElement dropdown options in node, ensure you're " +
+        'using jsdom-global or similar.',
+    );
+    return null;
```

Comment:
Is it worth verifying the console warning & null case in tests?

Reply:
No, because in order to get here you'd have to pass something that is an HTMElement type but then don't have HTMLElement defined. It would be really annoying to set up in tests and is a scenario that is unlikely to happen at all. But I had to return null to satisfy the ts compiler and the warning helps figure out why that is, in the off chance it ever somehow does happen.